satellite/analytics/service.go

```diff
@@ -11,7 +11,10 @@ import (
 )
 
 const (
-	eventAccountCreated = "Account Created"
+	eventAccountCreated     = "Account Created"
```

Comment:
Nit: I'm not sure why these constants are necessary?

CC: @mobyvb

Reply:
mainly so there is a source of truth at the top of the file rather than being mixed in throughout

Reply:
Yes, but currently it's being used only in one location, so there shouldn't be multiple-sources of truth problem in the first place.

Reply:
There are client-side events we still want to know about, such as when the customer generates gateway credentials in the satellite UI - in these cases, the client calls an endpoint on the satellite with the name of the event (e.g. "Credentials Created"). See https://review.dev.storj.io/c/storj/storj/+/4380
In the future maybe we could pass the event names through to the client, but I think that might actually end up being more confusing.